lwcomponent/component.go

```diff
@@ -255,12 +255,45 @@ type Component struct {
 	Name          string         `json:"name"`
 	Description   string         `json:"description"`
 	Type          Type           `json:"type"`
-	LatestVersion semver.Version `json:"version"`
+	LatestVersion semver.Version `json:"-"`
```

Comment:
We need this field.

Reply:
Please see `MarshalJSON` and `UnmarshalJSON` which properly handle this field.